srcpkgs/chrony/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'chrony'
 # When Updating: Please confirm the upstream config still refers to make_dirs
```

Comment:
Did you check this?

Reply:
I'm not sure how I can check this, but if I do a simple grep, I get the following output. So I assume that the upstream config still refers `make_dirs`.

![image](https://github.com/void-linux/void-packages/assets/58117792/58a5cf08-b880-4409-b217-970246ae547c)